pkg/signature/manifest.go

```diff
@@ -18,13 +18,14 @@ import (
 )
 
 const (
-	PublicKey               = "MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAETP8OUc6rTZHJs98X1aJfDIO0BXihHnSBDJhacdxZwk9RVzq28OIxQSVXXhD5ATEqWcNSWLnCG/GrZY9W2NfoMw=="
```

Comment:
Updating the key to reflect the new key post `AddOns` -> `Packages` rename which also resulted in a new key name.